Directory.Build.props

```diff
@@ -10,4 +10,45 @@
   <PropertyGroup Condition="'$(StabilizePackageVersion)' == 'true'">
     <StableVersion>$(VersionPrefix)</StableVersion>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <_hostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</_hostArch>
+    <BuildArchitecture>$(_hostArch)</BuildArchitecture>
+  </PropertyGroup>
+
+    <!-- Common properties -->
+  <PropertyGroup>
+
+    <__ProjectDir Condition="'$(__ProjectDir)'==''">$(MSBuildThisFileDirectory)</__ProjectDir>
+    <ProjectDir>$(__ProjectDir)\</ProjectDir>
```

Comment:
Just for reference, if it helps: https://learn.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2022#msbuild-ensuretrailingslash